scripts/vnet_route_check.py

```diff
@@ -237,10 +246,12 @@ def get_vnet_routes_from_asic_db():
 
     vnet_vrfs = get_vrf_entries()
     vnet_vrfs_oids = [vnet_vrfs[k] for k in vnet_vrfs]
+    vnet_vrfs_oids.append(default_vrf_oid) 
 
     vnet_intfs = get_vnet_intfs()
 
     vrf_oid_to_vnet_map = {}
+    vrf_oid_to_vnet_map[default_vrf_oid] = 'default_VRF'
```

Comment:
I don't see this being used. Can you confirm?

Reply:
This is used as a placeholder, as the data structure keeps track of the routes by means of the vnet interface.